sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ### New Features
 - Issue [17614](https://github.com/Azure/azure-sdk-for-java/issues/17614) : Adding support for AMQP Data types SEQUENCE 
   and VALUE. It support sending and receiving of only one AMQP Sequence at present.
```

Comment:
2 suggestions:
1. Put the text before issue number.
2. The text isn't consistent with line number 21's text. Suggest "Added support for ..."